### ISSUE TITLE
How do you measure compression quality?

### ISSUE THREAD

**Issue by avl-bsuir** [alexandru-dinu/cae#29]
Hi! 

What is your loss function and what metric do you use to evaluate quality of decompressed images?

Thank you,
Anton.

**Comment by alexandru-dinu** [comment]
Hello Anton,

[Mean Squared Error](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html) is used to compute the reconstruction error between the output and original image. https://github.com/alexandru-dinu/cae/blob/f0f60f768c78cf1d182c6778df6cad6d69712634/src/train.py#L57-L58

Currently, there is no other measure of quality. An in-depth evaluation and report is needed. 
It has been on my to-do list for quite some time now, but unfortunately did not get the chance to write it.

If you have any suggestions, please let me know. Also, feel free to submit pull requests.

Thank you,
Alex
